gazelle/pythonconfig/pythonconfig.go

```diff
@@ -425,3 +435,19 @@ func (c *Config) SetDefaultVisibility(visibility []string) {
 func (c *Config) DefaultVisibilty() []string {
 	return c.defaultVisibility
 }
+
+// SetTestFilePattern sets the file patterns that should be mapped to 'py_test' rules.
+// Will exit 1 if the pattern is invalid.
+func (c *Config) SetTestFilePattern(patterns []string) {
+	for _, p := range patterns {
+		if !doublestar.ValidatePattern(p) {
```

Comment:
Would this check be better in `configure.go` `case pythonconfig.TestFilePattern:`?

Reply:
I'd say it would be slightly more aligned with existing code to do to in a `case` as there are already some `invalid value` error, parsing etc and these are simple setters.

Reply:
👍  moved.